tests/conftest.py

```diff
@@ -160,6 +160,20 @@ def records_schema_v2():
         }
 
 
+@pytest.fixture(scope='session')
+def blobs_schema():
+    return {
+        'type': 'array',
+        'items': {
+                'type': 'object',
+                "propertyNames" : {
+                    "pattern": types.NAME_PATTERN
+                },
+                'additionalProperties': {"type": "string"}
```

Comment:
Is this preventing cardinality n values? (they are arrays of strings in JSON).

Reply:
Yes it is, I need to make the same change as https://github.com/openregister/conformance-test/pull/36